src/app/static/src/app/root.ts

```diff
@@ -42,13 +43,7 @@ export const emptyState = {
     stepper: initialStepperState
 };
 
-export const initialState = {
-    ...emptyState,
-    ...localStorageManager.getState()
-};
-
 export const storeOptions: StoreOptions<RootState> = {
-    state: initialState,
```

Comment:
This is irrelevant but not needed - the state gets overwritten by the modules straight away.